tests/evm_ds_test/tests/conftest.py

```diff
@@ -19,6 +19,7 @@
 import numpy
 import random as rand
 
+
```

Comment:
unnecessary

Reply:
it is coming from 'black', the formatting tool